x-pack/plugin/security/src/test/java/org/elasticsearch/xpack/security/authc/ApiKeyIntegTests.java

```diff
@@ -533,21 +539,62 @@ public void testInvalidateApiKeysOwnedByCurrentAuthenticatedUser() throws Interr
         verifyInvalidateResponse(noOfApiKeysForUserWithManageApiKeyRole, userWithManageApiKeyRoleApiKeys, invalidateResponse);
     }
 
-    public void testApiKeyAuthorizationApiKeyMustBeAbleToRetrieveItsOwnInformation() throws InterruptedException, ExecutionException {
+    public void testApiKeyAuthorizationApiKeyMustBeAbleToRetrieveItsOwnInformationButNotAnyOtherKeysCreatedBySameOwner()
+        throws InterruptedException, ExecutionException {
         List<CreateApiKeyResponse> responses = createApiKeys(2, null);
```

Comment:
I'd like this to be more explicit that it _doesn't_ have `manage_own_api_keys` (which is the functionality under test).
It's relying on the fact this method only gives `monitor`, but that's buried in the implementation of `createApiKeys` and could change.
Ideally this should run with the `"none"` cluster privilege.